studies/migrations/0054_new-analytics-permissions.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-10-23 16:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("studies", "0053_deprecate-old-eligiblity-form")]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="response",
+            options={
+                "base_manager_name": "related_manager",
+                "ordering": ["-demographic_snapshot__created_at"],
+                "permissions": (
+                    ("view_response", "View Response"),
+                    ("view_response_analytics_data", "View Response Analytics Data"),
```

Comment:
Can the human-readable name of this one also be clearer, e.g. View All Response Data in Analytics or View Responses from All Studies in Analytics